src/realm-settings/KeysProvidersTab.tsx

```diff
@@ -131,8 +134,37 @@ export const KeysTabInner = ({ components, refresh }: KeysTabInnerProps) => {
   };
 
   const onDragFinish = (itemOrder: string[]) => {
-    setItemOrder(["data", ...itemOrder.filter((i) => i !== "data")]);
+    setItemOrder(itemOrder);
     setLiveText(t("common:onDragFinish"));
+    const updateAll = components.map((component: ComponentData) => {
+      const componentToSave = { ...component };
+      componentToSave.providerDescription = undefined;
```

Comment:
An alternative approach for this could be:
```suggestion
      const componentToSave = { ...component, providerDescription: undefined };
```
Or
```suggestion
      const componentToSave = { ...component };
      delete componentToSave.providerDescription;
```